packages/llamaindex/src/readers/LlamaParseReader.ts

```diff
@@ -278,19 +280,29 @@ export class LlamaParseReader extends FileReader {
     fileContent: Uint8Array,
     fileName?: string,
   ): Promise<Document[]> {
-    // Creates a job for the file
-    const jobId = await this.createJob(fileContent, fileName);
-    if (this.verbose) {
-      console.log(`Started parsing the file under job id ${jobId}`);
-    }
+    let jobId;
+    try {
+      // Creates a job for the file
+      const jobId = await this.createJob(fileContent, fileName);
```

Comment:
don't create a new jobId
```suggestion
      jobId = await this.createJob(fileContent, fileName);
```